.github/scripts/install_straxen.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+cd straxen/straxen
+python setup.py install
```

Comment:
This won't work until we fix:
https://github.com/XENONnT/utilix/issues/12
right?

Reply:
It works on my fork...

Reply:
hmm okay, perhaps git is smarter than travis, travis could not find utilix at least in the base env:
https://travis-ci.org/github/XENONnT/base_environment/builds/743581784